zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -139,20 +141,32 @@ def channels2zerver_stream(slack_dir, realm_id, added_users):
             id=stream_id_count)
         zerver_stream.append(stream)
         added_channels[stream['name']] = stream_id_count
+       
+        # construct the recipient object and append it zerver_recipient
+        recipient = dict(
+             type_id=stream_id_count,
+             id=stream_id_count,
+             type=2)
+        zerver_recipient.append(recipient)
+        # TODO add recipients for private messages and huddles 
 
         # construct the subscription object and append it to zerver_subscription
         for member in channel['members']:
             sub = dict(
-                recipient=added_users[member],
+                recipient=stream_id_count,
```

Comment:
@rheaparekh thanks for spotting the subtle mistake ! Maybe a comment should be added that says it should be `added_users[member]` when the case is PM/huddle. In the newly added "TODO". Also, I write PM/huddle-related TODO as "TOODO" so that it is clear they are to be worked later rather than ASAP.